api/src/main/java/org/openmrs/module/reporting/definition/library/implementerconfigured/BaseImplementerConfiguredDefinitionLibrary.java

```diff
@@ -125,6 +123,8 @@ private void loadDefinitions() {
 							log.warn("Invalid serialized definition at " + libraryNameSuffix + " in " + file
 									.getAbsolutePath(), ex);
 						}
+					} else if (filename.endsWith(".groovy")) {
+						definition = (Definition) new GroovyHelper().parseClassFromFileAndNewInstance(file);
```

Comment:
the one thing I might still want to do here is use the Spring Application Context to populate any @Autowired properties in this file.